tests/plugins/magicline/manual/lineattribute.html

```diff
@@ -0,0 +1,67 @@
+<textarea cols="80" id="editor1" name="editor1" rows="10">
+  &lt;table border=&quot;1&quot; cellpadding=&quot;1&quot; cellspacing=&quot;1&quot; style=&quot;width: 100%; &quot;&gt;
+    &lt;tbody&gt;
+      &lt;tr&gt;
+        &lt;td&gt;This table&lt;/td&gt;
+        &lt;td&gt;is the&lt;/td&gt;
+        &lt;td&gt;very first&lt;/td&gt;
+        &lt;td&gt;element of the document.&lt;/td&gt;
+      &lt;/tr&gt;
+      &lt;tr&gt;
+        &lt;td&gt;We are still&lt;/td&gt;
+        &lt;td&gt;able to acces&lt;/td&gt;
+        &lt;td&gt;the space before it.&lt;/td&gt;
+        &lt;td&gt;
+        &lt;table border=&quot;1&quot; cellpadding=&quot;1&quot; cellspacing=&quot;1&quot; style=&quot;width: 100%; &quot;&gt;
+          &lt;tbody&gt;
+            &lt;tr&gt;
+              &lt;td&gt;This table is inside of a cell of another table.&lt;/td&gt;
+            &lt;/tr&gt;
+            &lt;tr&gt;
+              &lt;td&gt;We can type&amp;nbsp;either before or after it though.&lt;/td&gt;
+            &lt;/tr&gt;
+          &lt;/tbody&gt;
+        &lt;/table&gt;
+        &lt;/td&gt;
+      &lt;/tr&gt;
+    &lt;/tbody&gt;
+  &lt;/table&gt;
+
+  &lt;p&gt;Two succesive horizontal lines (&lt;tt&gt;HR&lt;/tt&gt; tags). We can access the space in between:&lt;/p&gt;
+
+  &lt;hr /&gt;
+  &lt;hr /&gt;
+  &lt;ol&gt;
+    &lt;li&gt;This numbered list...&lt;/li&gt;
+    &lt;li&gt;...is a neighbour of a horizontal line...&lt;/li&gt;
+    &lt;li&gt;...and another list.&lt;/li&gt;
+  &lt;/ol&gt;
+
+  &lt;ul&gt;
+    &lt;li&gt;We can type between the lists...&lt;/li&gt;
+    &lt;li&gt;...thanks to &lt;strong&gt;Magicline&lt;/strong&gt;.&lt;/li&gt;
+  &lt;/ul&gt;
+
+  &lt;p&gt;Lorem ipsum dolor sit amet dui. Morbi vel turpis. Nullam et leo. Etiam rutrum, urna tellus dui vel tincidunt mattis egestas, justo fringilla vel, massa. Phasellus.&lt;/p&gt;
+
+  &lt;p&gt;Quisque iaculis, dui lectus varius vitae, tortor. Proin lacus. Pellentesque ac lacus. Aenean nonummy commodo nec, pede. Etiam blandit risus elit.&lt;/p&gt;
+
+  &lt;p&gt;Ut pretium. Vestibulum rutrum in, adipiscing elit. Sed in quam in purus sem vitae pede. Pellentesque bibendum, urna sem vel risus. Vivamus posuere metus. Aliquam gravida iaculis nisl. Nam enim. Aliquam erat ac lacus tellus ac felis.&lt;/p&gt;
+
+  &lt;div style=&quot;border: 2px dashed green; background: #ddd; text-align: center;&quot;&gt;
+  &lt;p&gt;This text is wrapped in a&amp;nbsp;&lt;tt&gt;DIV&lt;/tt&gt;&amp;nbsp;element. We can type after this element though.&lt;/p&gt;
+  &lt;/div&gt;
+</textarea>
+
+<script>
+  CKEDITOR.replace( 'editor1', {
+  extraPlugins: 'magicline',
+  magicline_color: 'blue',
+  allowedContent: true
```

Comment:
I think instead of blocking ACF, you could just attach proper plugins to the editor, so you will have:

`@bender-ckeditor-plugins: wysiwygarea, basicstyles, elementspath, magicline,toolbar, table, list, magicline, div, horizontalrule`

and then editor config will need only `magicline_color: 'blue'`.